.github/workflows/release.yaml

```
@@ -54,6 +54,8 @@ jobs:
     steps:
       - name: Checkout git repository
         uses: actions/checkout@master
+      - name: rust version
+        run: rustup default 1.58.1
```

We should really standardize on using
```suggestion
      - name: Install Rust
        uses: actions-rs/toolchain@v1
        with:
          toolchain: 1.58.1
          default: true
```